### ISSUE TITLE
Issues running: source.fixAll.eslint

### ISSUE THREAD

**Issue by bjufre** [sublimelsp/LSP-eslint#48]
Hello, I'm trying to use this plugin in a project that I recently started working on and I follow every possible instruction but I don't seem to be able to run the `eslint --fix` option when saving a file; even when having the following setting in the `LSP`:

```json
"lsp_code_actions_on_save": {
    "source.fixAll.eslint": true,
},
```

Any ideas? 🤔 🙏

**Comment by rchl** [comment]
Information provided by `LSP: Troubleshoot server` and server logs would help to understand your issue.

See https://lsp.sublimetext.io/troubleshooting/#self-help-instructions for more info.

**Comment by rchl** [comment]
Also, run `sublime.log_commands(True)` in Sublime console and check what commands are triggered on saving a file.

If `lsp_save` is missing then that's your issue. It could be missing if you are using some kind of Vim-style plugin or using non-standard key bindings.

**Comment by bjufre** [comment]
@rchl you're completely right. It was because I had a custom key binding of `super+s` set to `save_all`.
Is there a way to have the `save_all` behaviour but still keep the `lsp_save`?

**Comment by rchl** [comment]
There is a `lsp_save_all` equivalent for `save_all`.

**Comment by bjufre** [comment]
FIXED EVERYTHING! Thank you!
